presto-main/src/test/java/io/prestosql/block/BlockAssertions.java

```diff
@@ -561,6 +562,15 @@ public static Block createLongDecimalSequenceBlock(int start, int end, DecimalTy
         return builder.build();
     }
 
+    public static Block createColorSequenceBlock(int start, int end)
```

Comment:
Let's use `value` and `positionCount` instead

Reply:
With `value` and `positionCount` would we create a block of repeated values ? I wanted to create a block of all different values to easily exceed the current distinct values count threshold in tests, that's why I used `start` and `end` as parameter.